src/abstract/BatchGovernor.sol

```diff
@@ -472,6 +472,7 @@ abstract contract BatchGovernor is IBatchGovernor, ERC712Extended {
         uint8 support_,
         string memory reason_
     ) internal virtual {
+        if (weight_ == 0) return;
```

Comment:
```suggestion
        if (weight_ == 0) revert ZeroVotes()
```

Reply:
Agree it should revert, but perhaps `ZeroVotingPower()`.